engine-tests/src/main/java/org/terasology/engine/integrationenvironment/jupiter/MTEExtension.java

```diff
@@ -163,14 +149,20 @@ private Object getDIInstance(Engines engines, Class<?> type) {
         }
     }
 
+    @SuppressWarnings("removal")  // TODO: replace UseWorldGenerator in modules
```

Comment:
Do we have issues for replacing `UseWorldGenerator` and `Dependencies` annotations in modules yet?
I would definitely be up with taking care of those tasks, but an issue documenting the migration path would be nice 🙂